src/main/webapp/resources/js/components/Header/main-navigation/components/AnnouncementsSubMenu.jsx

```diff
@@ -48,8 +47,24 @@ export function AnnouncementsSubMenu() {
     });
   }
 
-  const aMenu = (
-    <Menu className="t-announcements-submenu" theme={theme}>
+  return (
+    <Menu.SubMenu
+      theme="light"
```

Comment:
Same as above, let's use the theme from theme-utilities

Reply:
Updated